crowsnest/components/cam.py

```
@@ -0,0 +1,62 @@
+        component = utils.load_component(
+            self.parameters["mode"],
+            self.name,
+            config_section,
+            path="crowsnest.components.streamer",
+        )
+        if component is None or not isinstance(component, Streamer):
+            self.log_error(f"Tried to load a component that is not a Streamer!")
+            return
+        self.streamer: Streamer = component
+
+    def check_config_section(self, config_section: SectionProxy) -> bool:
```

Ruff (0.14.14)

Unused method argument: `config_section`

(ARG002)